### ISSUE TITLE
.

### ISSUE THREAD

**Issue by BoBeR182** [SpectralOps/keyscope#37]
Totally messed up and had my personal email attached to the author of the last 3 merges.
Not sure if this new merge request will rewrite history or just add 3 new commits, but it would be awesome if you folks could clean up my mistake.

https://github.com/SpectralOps/keyscope/pull/36

**Comment by BoBeR182** [comment]
I think you need to rebase from last known good commit at https://github.com/SpectralOps/keyscope/commit/d561c7f7fbce8424d495d414e7db229784639ddd thank you in advance

**Comment by BoBeR182** [comment]
I don't think it worked.

**Comment by BoBeR182** [comment]
On master `git rebase -i d561c7f `

make sure the following is set to drop first 3 commits
```
drop 413ad1c wakatime api (#35)
drop 8b74eb0 opsgenie api (#34)
drop b2ca75b hubspot api (#33)
pick a8c2d98 fix author history (#36) # empty
```
then just a `git push -f`
This will rewrite history
@jondot
